web/deploy/terraform/production/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_version = ">= 1.8.0, < 2.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+
+  backend "s3" {
+    bucket         = "${var.state_bucket_name}-${var.environment}"
+    key            = var.state_backend_key
+    region         = var.state_storage_region
+    dynamodb_table = "${var.state_table_name}-${var.environment}"
+    encrypt        = true
+  }
+}
+
+module "ec2" {
+  source        = "../modules/ec2/"
```

Comment:
This will need to be changed to a github URL with a reference to a tag after this PR is merged

Reply:
@smokestacklightnin why not leave it like this? What's the advantage of referencing the source as a GitHub URL?

Reply:
If the ec2 config changes from stage to prod, this is how we can keep track of those changes

Reply:
You mean the module changing? I'd say let's not overcomplicate things for now and keep the reference to the local module